methods/list/list.spec.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const algolib = require('../../index.js');
+
+describe.only('#list', function() {
+
+	it('should #get element from list', function() {
+		let list = new algolib.list();
+		list.push(5);
+		list.push(9);
+		list.push(8);
+		list.push(6);
+		list.pop();
+		list.pop();
+		list.pop();
+		list.pop();
+		list.pop();
```

Comment:
where you check in this test that `pop` get element from list?
it maybe return null or 'anything'